src/Compilers/VisualBasic/Test/Semantic/Binding/Binder_Expressions_Tests.vb

```diff
@@ -3041,7 +3037,7 @@ Class Program
             Dim x As Object = ""
             Color.M(x)
         Catch e As Exception
-            Console.WriteLine(e)
+            Console.WriteLine(e.GetType())
```

Comment:
I think there are two ways to handle this. The first is to think of this as a binding test, and check that the compiler correctly binds the code by running it and ensuring it has the correct runtime semantics. That is the current state of the code after the proposed change. I believe that satisfies the requirements of the purpose of this test and it is a reasonable place to take this code. It is possible that the code generated for this construct may change in the future - the library code could be inlined by the compiler, for example - and this test would still pass if it has the correct semantics.

The alternative is to replace or augment this test with an emit test that checks for the precise code generated for the constructs in question. Such tests are necessarily more delicate and expensive to maintain while the compiler and its libraries change in the future. I don't think that would be particularly valuable in this case, but it would satisfy the purpose of @AlekseyTs 's request.